olm/build.gradle.kts

```diff
@@ -58,13 +70,27 @@ kotlin {
         commonTest {
             dependencies {
                 implementation(kotlin("test"))
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:${kotlinCoroutines}-native-mt")
```

Comment:
`-native-mt` is deprecated. At least in this project.
```suggestion
                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:${kotlinCoroutines}")
```

Reply:
removed it.